employees/templates/employees/author_report_list.html

```diff
@@ -46,4 +47,9 @@
         integrity="{% staticinline 'employees/scripts/modalsFunctionality.js' encode="sri" %}"
         crossorigin="anonymous">
     </script>
+    <script
+        src="{% static 'users/scripts/hidePopupMessage.js' %}"
+        integrity="{% staticinline 'users/scripts/hidePopupMessage.js' encode="sri" %}"
```

Comment:
Are you sure you did here write quotes? First after equal sign is `"` then again you're using it after next equal sign with encode parameter. Shouldn't be there single quote sign `'`?

Reply:
More proper would be a single quote sign `' ` for sure. I changed that. However, I believe that as it is surrounded by `{%` and `%}` it doesn't affect the correct work.